lib/droplet_kit/client.rb

```diff
@@ -23,7 +23,7 @@ def self.resources
         cdns: CDNResource,
         certificates: CertificateResource,
         droplets: DropletResource,
-        kubernetes: KubernetesResource,
+        kubernetes_clusters: KubernetesClustersResource,
```

Comment:
Should this be `kubernetes_cluster: KubernetesClusterResource`, following the singular format of rest of the resources?

Reply:
looks like the other resources are plural, but the file/class names are singular. @joonas